### ISSUE TITLE
Possible new way for storage

### ISSUE THREAD

**Issue by Dark-Matter1** [benjamin-weatherill/CSI-280-RPG_Management#49]
### New Idea
So I heard about an alternative way to store things which is very simple, and since this isn't like a competitive game, and more a storage/organizer/simulation we don't need to store anything complex. I propose we use simple text as storage. We might still be able to use a JSON format, but we could literally just write to like a string and when the save button is pressed, we write it to a file and give that to the user, then when the user wants to load data, we just parse the file according to a schema we determine on save or whatever. I think this would simplify the issue of saving greatly. Plus if we are just saving to a string or something we could also use `doccument.getElementById("").innerHTML` to grab the text before we save it.

@drewbarnett @justinwillis145 @Andrew-Vinci #6

**Comment by Andrew-Vinci** [comment]
Yeah that sounds perfect and I think it's a good idea. It sounds like that would be the easiest way to store information locally and give us some sort of storage for different characters.
